en/docs/references/choreo-expression-syntax.md

```diff
@@ -0,0 +1,344 @@
+# Choreo Expression Syntax
+
+Choreo expressions comprise a powerful expression syntax based on the Ballerina language for function invocations, declaring a variable using literals and tuples, typecasting, etc. This document presents you the common expressions that you can use while developing Choreo applications.
+
+## Supported Literals 
+
+### String
+
+A string is simply a sequence of characters. It is required to use double quotes (")  to mark the boundary between the starting and ending characters.
+
+#### Examples
+
+- ```
+  "Colombo"
+  ```
+- ```
+  "12345"
+  ```
+- ```
+  "Ballerina is a programming language intended for network-distributed applications."
+  ```
+
+### Signed Integer
+
+Using this type, you can give numbers as an input to an expression-supported field.  You have the option to use `+` or `-` characters in the front to indicate the sign.  You can also define an integer by using only numerical characters without any delimiters. 
+
+#### Examples
+
+- ```
+  1234
+  ```
+- ```
+  -10
+  ```
+- ```
+  +2020
+  ```
+
+#### Decimal Floating-Point Number
+
+A number with a decimal point falls under this data type. You have the option to use `+` or `-` characters in the front to indicate the sign.
+
+#### Examples
+
+- ```
+  0.3353
+  ```
+- ```
+  2044.24
+  ```
+- ```
+  -8593.992
+  ```
+- ```
+  +950.930
+  ```
+- ```
+  10.0
+  ```
+
+### Boolean Value
+
+This data type has one of two possible values, true and false.
+
+#### Examples
+
+- ```
+  true
+  ```
+- ```
+  false
+  ```
+
+## Accessing Variables
+
+### Using the variable value
+
+Variables keep values in memory for later access. In an expression syntax-supported input field, you can refer to these variables directly using the variable name.
+
+#### Examples
```

Comment:
Should we change this to `Example` as we are showing only one example below?

Reply:
Done

Reply:
Check all the places. There are few other places.